tests/test_workflow_runner.py

```diff
@@ -83,21 +83,23 @@ def test_run_setup(config: Config, tmp_path: Path):
     assert runner.dpath_pipeline_work.exists()
 
 
-def test_run_cleanup(tmp_path: Path):
+@pytest.mark.parametrize("keep_workdir", [True, False])
+def test_run_cleanup(tmp_path: Path, keep_workdir):
```

Comment:
This looks good but we should also include another test for different values of `n_success` and `n_total` (this is also what CodeCov is complaining about. See e.g. [this test](https://github.com/nipoppy/nipoppy/blob/main/tests/test_workflow_pipeline.py#L616) which sets `n_success` and `n_total` to different values with different expected results for each pair